src/reanimated2/initializers.ts

```diff
@@ -29,36 +29,38 @@ function valueUnpacker(objectToUnpack: any, category?: string): any {
     workletsCache = global.__workletsCache = new Map();
     handleCache = global.__handleCache = new WeakMap();
   }
-  if (objectToUnpack.__workletHash) {
-    let workletFun = workletsCache.get(objectToUnpack.__workletHash);
+  const workletHash = objectToUnpack.__workletHash;
+  if (workletHash) {
```

Comment:
I know it's highly unlikely but this won't work properly if `workletHash` equals 0. Can we add `!== undefined` here?